### PR TITLE
DDF-5998 Upgraded to GIB 3.10.0

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Upgrades GIB to see if we get some performance improvements

#### Who is reviewing it? 
@LinkMJB 
@TonyMorrison 
@brhumphe 

#### Select relevant component teams: 
@codice/build 
@codice/continuous-integration 

#### Ask 2 committers to review/merge the PR and tag them here.
@mcalcote 
@rzwiefel 

#### What are the relevant tickets?
Fixes: #5998 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/5999)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

previous build aborted due to time limit

**Comment by shaundmorris** [comment]
build now

File: Jenkinsfile [hunk 1]
```diff
@@ -68,8 +68,8 @@ pipeline {
                 // TODO: Maven downgraded to work around a linux build issue. Falling back to system java to work around a linux build issue. re-investigate upgrading later
                 // -DskipITs is temporary to skip all the tests that were failing at the time. See https://github.com/codice/ddf/issues/5777
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                    sh 'mvn install -B -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```
Comment (LinkMJB) [inline Jenkinsfile:71]: Only comment, is we should restore `$DISABLE_DOWNLOAD_PROGRESS_OPTS` prior to merging this PR

Comment (shaundmorris) [reply Jenkinsfile]: Good call! Added this back in.

**Comment by shaundmorris** [comment]
build now

**Pull request merged**